README.md

```diff
@@ -58,6 +58,7 @@ These are args currently supported:
 - `rbd <args>` : Call a 'rbd' CLI command with arbitrary args
 
 - `mons` : Print mon endpoints
+  - `restore-quorum <mon-name>` : Restore the mon quorum to a single mon since quorum was lost with the other mons
```

Comment:
```suggestion
  - `restore-quorum <mon-name>` : Restore the mon quorum to a single mon since quorum was lost with the other mons
```
is this right? `to a single mon`

Reply:
I'll rephrase it for clarity